source/developers-guide/product-export/index.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: Extending product exports
+github_link: developers-guide/product-export/index.md
+shopware_version: 5.3.0
+tags:
+  - product
+  - export
+  - developers
+  - beginner
+indexed: true
+group: Developer Guides
+subgroup: General Resources
+menu_title: Product exports
+menu_order: 600
+---
+
+<div class="toc-list"></div>
+
+## Introduction
+This article will show examples for extending product exports in shopware
+
+## Modify the article variables
+The `Shopware_Modules_Export_ExportResult_Filter_Fixed` event can be used to modify the exported  article data. One use case would be adding new variables to every article or edit one of the existing variables. Even more you could create your own filter to remove articles which meet certain criteria. Here is a basic example how to use that event in a plugins bootstrap:
```

Comment:
... the exported product data. A use case could be adding new variables to every product or ... 

You can also create your own filter to remove products which meet certain criteria. 

... in a plugin base file 

(bootstrap sound too much like the old plugin system)